libs/react-components/community/src/components/form/file-upload/file-upload.tsx

```diff
@@ -69,6 +69,14 @@ export interface FileUploadProps extends FormLabelProps {
    * @default false
    */
   disabled?: boolean;
+  /**
+   * Size limit in bytes
+   */
+  sizeLimit?: number;
```

Comment:
Maybe we can use size in  MB instead of bytes here? So the applications can pass just `sizeLimit={5}`, not `sizelimit={5242880}` (maxAllowedSize = 5 * 1024 * 1024)